packages/tokens/src/bin/Utils/resolve.ts

```diff
@@ -0,0 +1,5 @@
+export const resolve = (object: any, path: string, separator: string = ".") => {
```

Comment:
Instead of any, we should use at least `Object` WDYT ?

Reply:
I'm going to use `Record<string, any>` instead. Because `Object` causes errors like " type 'string' can't be used to index type 'Object'."